dateparser/utils/__init__.py

```diff
@@ -133,6 +135,24 @@ def apply_timezone_from_settings(date_obj, settings):
     return date_obj
 
 
+def get_last_day_of_month(year, month):
+    return calendar.monthrange(year, month)[1]
+
+
+def set_correct_day_from_settings(date_obj, settings, last_day=None, current_day=None):
+    """ Set correct day attending the `PREFER_DAY_OF_MONTH` setting."""
```

Comment:
any better idea for the naming of this function?

Reply:
https://github.com/scrapinghub/dateparser/pull/611#discussion_r368165205